File: jobs/2026-06-01__10-59-51/shared_team_memory_py__yQqhWCb/artifacts/user/myproject/test_search5.py
import os
from alchemyst_ai import AlchemystAI

client = AlchemystAI()
def test_filter(f):
    try:
        res = client.v1.context.search(
            minimum_similarity_threshold=0.0,
            similarity_threshold=1.0,
            query="test",
            body_metadata=f
        )
        print(f, len(res.contexts))
    except Exception as e:
        pass

test_filter({"$filter": {"file_name": "memory_test-session-1780301813765"}})
test_filter({"filters": {"file_name": "memory_test-session-1780301813765"}})
test_filter({"session_id": "test-session-1780301813765"})
test_filter({"filters": {"session_id": "test-session-1780301813765"}})

File: jobs/2026-07-04__06-18-47/alchemyst_context_arithmetic_ts__YAuB6VY/artifacts/user/myproject/test_search5.js
const AlchemystAI = require('@alchemystai/sdk').default;
async function go() {
  const c = new AlchemystAI({ apiKey: process.env.ALCHEMYST_AI_API_KEY, logLevel: 'debug' });
  try {
    const r = await c.v1.context.search({
      query: 'engineering notes',
      similarity_threshold: 0.1,
      minimum_similarity_threshold: 0.1,
      scope: 'internal',
      body_metadata: {
        groupName: ['eng', 'v1']
      }
    });
    const ctxs = (r && r.contexts) || [];
    console.log('count=' + ctxs.length);
  } catch (e) {
    console.error('ERR', e.status, (e.message||'').slice(0,200));
  }
}
go();
